src/utils/chart.ts

```diff
@@ -125,3 +127,73 @@ export const getFeeRateSamples = (feeRates: Array<FeeRateTracker.TransactionFeeR
 
   return samples
 }
+
+type AssertIsArray = (
+  value: EChartOption.Tooltip.Format | EChartOption.Tooltip.Format[],
+) => asserts value is EChartOption.Tooltip.Format[]
+
+export const assertIsArray: AssertIsArray = (value: EChartOption.Tooltip.Format | EChartOption.Tooltip.Format[]) => {
+  const isArray = Array.isArray(value)
+  if (!isArray) {
+    throw new Error('value is not an array')
```

Comment:
The message is not clear, it should express the expectation, such as `Value is expected to be an array, but got a {real type}`